src/Microsoft.DocAsCode.Dfm/Rules/DfmIncludeBlockRule.cs

```diff
@@ -7,15 +7,78 @@ namespace Microsoft.DocAsCode.Dfm
     using System.Text.RegularExpressions;
 
     using Microsoft.DocAsCode.MarkdownLite;
+    using Microsoft.DocAsCode.MarkdownLite.Matchers;
 
     public class DfmIncludeBlockRule : IMarkdownRule
     {
+        private static readonly Matcher _IncludeMatcher =
+            Matcher.WhiteSpacesOrEmpty +
```

Comment:
Not same with regex, but I think it should be allowed.

---
In reply to: [107637464](https://github.com/dotnet/docfx/pull/1472#discussion_r107637464) [](ancestors = 107637464)